File: cpp/medidas.cpp
#include <iostream>
#include <iomanip>
#include <math.h>

using namespace std;

int main() {

    double A, B, C, quadrado, triangulo, trapezio;

    cout << "Digite a medida A: ";
    cin >> A;

    cout << "Digite a medida B: ";
    cin >> B;

    cout << "Digite a medida C: ";
    cin >> C;

    quadrado = pow(A, 2.0);

    cout << fixed << setprecision(4);
    cout << "AREA DO QUADRADO = " << quadrado << endl;

    triangulo = (A * B) / 2;

    cout << "AREA DO TRIANGULO = " << triangulo << endl;

    trapezio = ((A + B) * C) / 2;

    cout << "AREA DO TRAPEZIO = " << trapezio << endl;

	return 0;
}

File: python/medidas.py
import math

a: float
b: float
c: float
quadrado: float
triangulo: float
trapezio: float

a = float(input("Digite a medida A: "))
b = float(input("Digite a medida B: "))
c = float(input("Digite a medida C: "))

quadrado = math.pow(a, 2.0)
triangulo = (a * b) / 2.0
trapezio = ((a + b) * c) / 2.0

print(f"AREA DO QUADRADO = {quadrado:.4f}")
print(f"AREA DO TRIANGULO = {triangulo:.4f}")
print(f"AREA DO TRAPEZIO = {trapezio:.4f}")
